formats/json/commonTest/src/kotlinx/serialization/json/JsonImplicitNullsTest.kt

```diff
@@ -0,0 +1,13 @@
+package kotlinx.serialization.json
+
+import kotlinx.serialization.*
+
+class JsonImplicitNullsTest: AbstractJsonImplicitNullsTest() {
+    override fun <T> Json.encode(value: T, serializer: KSerializer<T>): String {
```

Comment:
we have similar functions in `JsonTestBase`. can we reuse them?

Reply:
In `JsonTestBase` these functions are inline, but they must be overridden for each encoder/decoder type.